client/src/legacy/TinyMCE_sslink.js

```diff
@@ -131,14 +131,24 @@ jQuery.entwine('ss', ($) => {
       return Promise.resolve();
     },
 
-    buildAttributes(data) {
-      const anchor = data.Anchor && data.Anchor.length ? `#${data.Anchor}` : '';
-      const href = `${data.Link}${anchor}`;
+    /**
+     *
+     * @param {string|undefined} Anchor
+     * @param {string} Link
+     * @param {boolean} TargetBlank
+     * @param {string} Description
+     * @returns {{href: string, title: string, target: string}}
+     */
+    buildAttributes({ Anchor, Link, TargetBlank, Description }) {
```

Comment:
Minor thing, though I'd make the argument order { Link, Description, Anchor, TargetBlank } so that it matches the order of the form elements on the frontend